test/test_backend/db/tst_config.py

```diff
@@ -0,0 +1,47 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+
+TODO: When done: Tst -> Test
+
+todo: would be nice to have a test to make sure nothing in STANDALONE_TABLES doesn't exist in DB, for cleanliness
+"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+# noinspection DuplicatedCode
+THIS_DIR = Path(os.path.dirname(__file__))
+PROJECT_ROOT = THIS_DIR.parent.parent.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+
+from backend.db.config import DERIVED_TABLE_DEPENDENCY_MAP, STANDALONE_TABLES
+from backend.db.utils import list_schema_objects
+
+
+class TestBackendDbConfig(unittest.TestCase):
```

Comment:
## Add `TestBackendDbConfig`
Added this test and now it passes after doing these other updates. Further discussion in https://github.com/jhu-bids/TermHub/pull/577#discussion_r1461297206, although most of those concerns have now been resolved. Any remaining concerns have had GH issue created.